tibco/src/test/resources/jndi.properties

```diff
@@ -0,0 +1,14 @@
+# This files is used for a lookup towards Tibco EMS before the definite connection is established.
```

Comment:
```suggestion
# This file is used for a lookup towards Tibco EMS before the definite connection is established.
```

Reply:
Dit is een testbestand aangeleverd door de klant. Commentaar op de inhoud hierop is niet zo relevant...